src/firebaseConfig.ts

```diff
@@ -167,6 +167,7 @@ export type EmulatorsConfig = {
     host?: string;
     port?: number;
   };
+  single_project_mode?: boolean;
```

Comment:
I don't think this should be underscore_case - generally variables are camelCase

Reply:
Updated to singleProjectMode